libvast/vast/system/status.hpp

```diff
@@ -138,10 +138,10 @@ void collect_status(
   caf::settings& s, std::string_view key) {
   collect_status(
     rs, timeout, verbosity, responder,
-    [key, &s](caf::settings& response) {
+    [key = std::string{key}, &s](caf::settings& response) {
```

Comment:
There might be another minor cleanup possible: Do we actually have to use the explicit conversion to `std::string_view` on lines 142 and 145?